native/src/components/NearbyCities.tsx

```diff
@@ -24,13 +24,13 @@ const NearbyMessage = styled.Text`
   padding-top: 15px;
 `
 
-type Props = {
+type NearbyCitiesGroupProps = {
```

Comment:
The file name doesn't match the component name here.

Reply:
Yes, I think it should be renamed to `NearbyCities`.

Reply:
yup